### ISSUE TITLE
Native Notebooks - Different Markdown Icons

### ISSUE THREAD

**Issue by claudiaregio** [microsoft/vscode#115340]
Markdown symbol for Ctrl + Shift + O Navigation does not match the markdown icon in outline

![image](https://user-images.githubusercontent.com/19276747/106194983-6b509880-6164-11eb-9dd5-da6c88ffa79e.png)

![image](https://user-images.githubusercontent.com/19276747/106194954-5b38b900-6164-11eb-9a36-96bdd216ca9f.png)

**Comment by claudiaregio** [comment]
Same for when you select the language picker in a cell
![image](https://user-images.githubusercontent.com/19276747/106937970-3f3c9680-66d3-11eb-96fc-8f864195e911.png)

**Comment by claudiaregio** [comment]
cc/ @misolori

**Comment by misolori** [comment]
@jrieken I think at the minimum the outline in the view + symbols should match?

**Comment by jrieken** [comment]
Yeah, tho it depends on the icon theme... The challenge is that quick is pretty bad with mixed icons, e.g icons from the codicon font and from themes
